README.rst

```diff
@@ -0,0 +1,75 @@
+=============
+ResearchMate
+=============
+
+
+Overview
+========
+
+ResearchMate is an open-source Python package to manage and interact with ongoing researches.
+
+The main goal is establishing a Django Application that offers different views and functionality
+that are relevant for every-day use by mostly-academic researchers.
+
+* Free software: MIT License
+
+Getting Started
+================
+
+These instructions will get you up and running on your local machine.
+
+Prerequisites
+==============
+
+In order to run ResearchMate you should install:
+
+* Download and Install `Vagrant`_.
+* Download and Install `Virtual Box`_.
+
+Run the project on your local machine
+======================================
+A step by step series that tell you how to get Hotails up and running:
+
+1. Clone the repository
+2. Open command prompt in Hotails directory
+3. Spin the environment using vagrant up
```

Comment:
wrap the vagrant up to be code sample using ``
The result will be `vagrant up`

Reply:
Done.